app/src/main/java/com/infomaniak/mail/utils/WebViewUtils.kt

```diff
@@ -130,21 +131,26 @@ class WebViewUtils(context: Context) {
             jsBridge = JavascriptBridge(onWebViewFinishedLoading = onWebViewFinishedLoading)
         }
 
-        fun WebSettings.setupThreadWebViewSettings() {
+        private fun WebSettings.setupDefaultWebViewSettings() {
```

Comment:
Maybe we should use the term "common" over "default". These settings are not "default" per say. If we create a 3rd webview somewhere but don't need some of these settings we should not default to them without thinking just because they're the "default" ones